torchdynamo/config.py

```diff
@@ -99,12 +99,16 @@
     "torch._decomp",
 }
 
+# Debug Flag to try minifier at different stages. Possible values are {None, "aot", "dynamo"}
+# None - Minifier is switched off
+# dynamo - Runs minifier on the TorchDynamo produced graphs, if compilation fails
+# aot - Runs minifier on the Aot Autograd produced graphs, if compilation fails
+repro_after = os.environ.get("REPRO_AFTER", None)
 # Compiler compilation debug info
-# 0: Nothing printed out when compilation fails
-# 1: Dump the graph out to repro.py if compilation fails
-# 2: Dumps the graph out to minify_repro.py with a minifier if compilation fails
-# 3: Always dumps the last graph ran out to minify_repro.py, useful for segfaults/irrecoverable errors
-repro_level = int(os.environ.get("COMPILER_REPRO_LEVEL", 0))
+# 1: Dumps the original graph out to repro.py/repro.tar.gz if compilation fails
+# 2: Either 1) Dumps the original graph out to minify_repro.py with a minifier if compilation fails
+#        or 2) Minifies and dumps the minified graph to repro.py/repro.tar.gz if compilation fails
+repro_level = int(os.environ.get("REPRO_LEVEL", 2))
```

Comment:
```suggestion
repro_level = int(os.environ.get("TORCHDYNAMO_REPRO_LEVEL", 2))
```
I worry about colliding with names used by other projects.